.github/requirements/conda-env-macOS-ARM64

```diff
@@ -8,9 +8,10 @@ dataclasses=0.8
 pip=22.2.2
 six=1.16.0
 pillow=9.2.0
-libuv=1.39.0
+libuv=1.40.0
```

Comment:
I realize that this version is wrong and should be `1.40.0`, same as the one from `conda-env-macOS-X64`.  Using 1.39.0 here causes a conda dependency conflict